src/main/kotlin/com/atlassian/performance/tools/report/api/CohortsReporter.kt

```diff
@@ -0,0 +1,60 @@
+package com.atlassian.performance.tools.report.api
+
+import com.atlassian.performance.tools.io.api.ensureParentDirectory
+import com.atlassian.performance.tools.report.MeanAggregator
+import com.atlassian.performance.tools.report.api.result.Stats
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.csv.CSVPrinter
+import org.apache.logging.log4j.LogManager
+import java.io.File
+import java.time.Duration
+
+/**
+ * Summarizes per cohort statistical results.
+ */
+class CohortsReporter(
```

Comment:
"Reporter" is vague. `DistributionComparison`, `MeanLatencyChart` and `EditedIssuesReport` also report on multiple cohorts. Concretely, both `DataReporter` and `CohortsReporter` summarize the stats in a CSV table.

Some ideas to make it more concrete: `CohortsTable`, `MultiCohortStats`, `CohortsCsvStats`

Reply:
`CohortStatsSummary`? Because it "Summarizes"

Reply:
Sounds good